### DISCUSSION TITLE
HOW to use nonott

### DISCUSSION THREAD

**Discussion by Piceeeeeee** [roleoroleo/yi-hack-Allwinner-v2#911] [Q&A]
Hello, i have a yi home 1080p, with V5 hack on it, i want to be able to use the speaker but i dont know how, from my undestanding, i have to copy the binny file nonott in yihach/bin and copy the chare directory in usr, i have done that but now what? I appreciate any feedback

**Comment by roleoroleo** [comment]
You can't use speaker features with v5.
There is no support for audio in.
